src/index.js

```diff
@@ -1,34 +1,146 @@
+};
+
+const changeBackIcons = () => {
+  document.querySelectorAll('.label-text').forEach((button) => button.addEventListener('focusout', (event) => {
+    event.preventDefault();
+    const { id } = button;
+
+    const indexOldElement = 'ellipsis-vertical-'.concat(id.charAt(id.length - 1));
+    const oldElement = document.getElementById(indexOldElement);
+    const newElement = document.getElementById(`${id.charAt(id.length - 1)}`);
+    oldElement.style = 'display:block';
+    newElement.style = 'display:none';
+  }));
+};
+
+const showItems = () => {
+  toDoListHtml.innerHTML = '';
+  for (let i = 0; i < tasks.length; i += 1) {
+    toDoListHtml.innerHTML
+    += `
+    <li class="list">
+      <div class="task-item">
+        <span class="check-square"><i class="fa-regular fa-square"></i></span>
+        <span class="label-text" id="label-text-${i}"><input class="task-paragraph" type="text" value="${tasks[i].description}"></span>
+      </div>
+      <button class="trash-btn" id="${i}"><i class="fa-regular fa-trash-can"></i></button>
+      <span class="ellipsis-vertical" id=ellipsis-vertical-${i}><i class="fa-solid fa-ellipsis-vertical"></i></span>
+    </li>
+    `;
+  }
+  taskList.appendChild(toDoListHtml);
+
+  addTasksLocalStorage(tasks);
+  remove();
+  changeIcons();
+  changeBackIcons();
+};
+
+const warningTag = document.querySelector('.value-warning');
+
+const defineWarning = () => {
+  const warning = 'The value you enter should not be empty!';
+  warningTag.innerText = warning;
+  warningTag.style = 'display:block';
+};
+
+const resetWarning = () => {
+  warningTag.style = 'display:none';
+};
+
+taskItem.addEventListener('submit', (event) => {
+  event.preventDefault();
+
+  if (task.value.length > 0) {
+    tasks.push({ description: task.value, completed: false, index: tasks.length });
+    counter += 1;
+
+    toDoListHtml.innerHTML
+    += `
+    <li class="list">
+      <div class="task-item">
+        <span class="check-square"><i class="fa-regular fa-square"></i></span>
+        <span class="label-text" id="label-text-${counter}"><input class="task-paragraph" type="text" value="${tasks[counter].description}"></span>
+      </div>
+      <button class="trash-btn" id="${counter}"><i class="fa-regular fa-trash-can"></i></button>
+      <span class="ellipsis-vertical" id=ellipsis-vertical-${counter}><i class="fa-solid fa-ellipsis-vertical"></i></span>
+    </li>
+    `;
+    taskList.appendChild(toDoListHtml);
+    task.value = '';
+
+    addTasksLocalStorage(tasks);
+    remove();
+    changeIcons();
+    changeBackIcons();
+  } else {
+    defineWarning();
+    setTimeout(resetWarning, 3000);
+  }
+});
+
+window.addEventListener('load', (event) => {
+  if (
+    localStorage.getItem('storedBookData') !== null && localStorage.getItem('storedBookData') !== '[]'
+  ) {
+    event.preventDefault();
+    tasks = JSON.parse(localStorage.getItem('storedBookData'));
+    counter = tasks.length - 1;
+    showItems();
+  }
+});
```

Comment:
- Kindly make sure that your CRUD features are inside of a new JavaScript file. This is part of the requirements of the project. :pray: :pray: 
![image](https://user-images.githubusercontent.com/98363075/199865821-9a96621d-12e7-43bc-b446-38bc6cded941.png)

- Kindly make sure that each object's index is equal to the length of the new array. It means that the index should start at 1, not at 0. This is a requirement of the project. :+1: :smiley: :smiley: 
![image](https://user-images.githubusercontent.com/98363075/199866100-1f40c991-a32f-4db1-a66d-2dcae9260d56.png)